LICENSE

```diff
@@ -1,4 +1,6 @@
-Copyright (c) 2013-2014 Will Thames <will@thames.id.au>
+Copyright (c) 2013-2018 Will Thames <will@thames.id.au>
+Copyright (c) 2018 Ansible by Red Hat
```

Comment:
@abadger1999 is what I've done here correct?

Reply:
I'm not sure whether this should be added prior to any code changes actually generated by Red Hat...

Reply:
Red Hat have made contributions to ansible-lint already, so I'm fine with this.